sigridci/sigridci.py

```diff
@@ -69,8 +69,10 @@ def parseTarget(target):
     parser.add_argument("--exclude", type=str, default="", help="Comma-separated list of files/directories to exclude.")
     parser.add_argument("--include", type=str, default="", help="Comma-separated list of files/directories to include.")
     parser.add_argument("--showupload", action="store_true", help="Logs the contents of the upload published to Sigrid.")
-    parser.add_argument("--include-history", action="store_true", help="Publish repository history to Sigrid.")
     parser.add_argument("--sigridurl", type=str, default="https://sigrid-says.com", help=SUPPRESS)
+    # This option is now enabled by default, using an anonymized history,
+    # but we leave it here to avoid breaking people's configuration.
```

Comment:
That is actually not entirely true. The reason for leaving the argument is actually so that people don't get a "unknown argument —include-history" error.

Reply:
But wouldn't it be a good idea to have an escape hatch, just in case this needs to run, if even temporarily, in an environment where `git` is unavailable and/or cannot be used? Of course, I can think of a hundred thousand more things for which one day an escape hatch might be handy, but this one we already have, so we can as well keep it.